helpdesk_mgmt_fieldservice/models/fsm_order.py

```diff
@@ -0,0 +1,58 @@
+# Copyright (C) 2019 - TODAY, Open Source Integrators
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl).
+
+import logging
+
+from odoo import api, fields, models
+
+_logger = logging.getLogger(__name__)
+
+
+class FSMOrder(models.Model):
+    _inherit = 'fsm.order'
+
+    ticket_id = fields.Many2one('helpdesk.ticket', string='Ticket',
+                                track_visibility='onchange')
+
+    def action_complete(self):
+        res = super().action_complete()
+        if self.ticket_id:
+            open_fsm_orders_count = self.env['fsm.order'].search_count(
+                [('ticket_id', '=', self.ticket_id.id),
+                 ('stage_id.is_closed', '=', False)])
+
+            if self.ticket_id.stage_id.closed:
+                return res
+            elif open_fsm_orders_count == 0:
+                view_id = self.env.ref(
+                    'helpdesk_fieldservice.fsm_order_close_wizard_view_form'
```

Comment:
external id wrong:
helpdesk_**mgmt**_fieldservice

Reply:
Thank you. I'll review